### ISSUE TITLE
Payload items aren't sent on server?

### ISSUE THREAD

**Issue by radicaled** [rollbar/rollbar.js#323]
Trying to upgrade a project to the new universal / isomorphic rollbar.js. Works fine in the client, but in the server it seems that extra payload info (our custom fields being sent with exceptions) aren't going out with the error. Is there a cavaet when sending payload info on the server using 2.x or is this just a bug?

**Comment by rokob** [comment]
I'll look into this, there is no caveat to using this on the server like that so it would be a bug.

**Comment by rokob** [comment]
I can't see any reason why you would be seeing this, I have a test app and logging errors on the server with an extra custom object results in the custom data showing up in Rollbar. Can you give me a little more info on what your actual log call looks like?

**Comment by radicaled** [comment]
I've spent a bit looking at this on my side and I've discovered my actual problem: Rollbar on the server doesn't appear to be acknowledging my _configuration's payload object_, which in turn caused me to check the wrong test message and think that payload information wasn't being sent.

This is my server config:

```javascript
const ROLLBAR_CONFIG = {
  accessToken: '[CENSORED]',
  captureUncaught: true,
  captureUnhandledRejections: true,
  enabled: true,
  payload: {
    environment: 'production'
  }
};
```

It's a cut-and-paste of my client-side config. Regardless of what `payload.environment` is set as, it always sends out errors under the "development" environment. This configuration object works fine on the client.
